ct.sh

```diff
@@ -34,6 +34,7 @@ export CC_x86_64_fortanix_unknown_sgx=clang-11
 export CC_aarch64_unknown_linux_musl=/tmp/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc
 export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_LINKER=/tmp/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc
 export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_MUSL_RUNNER=qemu-aarch64
+export CARGO_INCREMENTAL=0
```

Comment:
Why?

Reply:
Since I turn on the `cache` for cargo in travis CI, I think INCREMENTAL build need to be turn off to avoid increase of `target`   size